src/app/models/form.server.model.ts

```diff
@@ -577,6 +577,24 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     )
   }
 
+  // Deletes specified form logic.
+  FormSchema.statics.deleteFormLogic = async function (
+    this: IFormModel,
+    form: IPopulatedForm,
```

Comment:
do you need to pass the entire form? would the form ID be sufficient? this would also make this function more extensible because it can be called without having to retrieve the entire populated form

Reply:
edited